modules/table/src/main/java/org/apache/ignite/internal/table/distributed/StorageUpdateHandler.java

```diff
@@ -284,8 +349,8 @@ public void handleWriteIntentRead(UUID txId, RowId rowId) {
     }
 
     /**
-     * Switches write intents created by the transaction to regular values if the transaction is committed
-     * or removes them if the transaction is aborted.
+     * Switches write intents created by the transaction to regular values if the transaction is committed or removes them if the
+     * transaction is aborted.
```

Comment:
Artifacts of auto-formatting the file I guess?
I don't like when people add them. You didn't change the content of the javadoc, why committing it then?